composer.json

```diff
@@ -21,11 +21,12 @@
     "require": {
         "php": "^8.0",
         "guzzlehttp/guzzle": "^7.4",
-        "illuminate/support": "^8.0 || ^9.0",
-        "nesbot/carbon": "^2.58"
+        "nesbot/carbon": "^2.58",
+        "ext-simplexml": "*"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^3.5",
+        "illuminate/support": "^9.34",
```

Comment:
as `9.34` isn't really required but `^9.0` works as well I would go for that constraint.
PS: if it's only for the `collect()` method it would be [`illuminate/collections`](https://github.com/illuminate/collections) instead of the whole `illuminate/support` package.

Reply:
I will work on it and send a new PR

Reply:
Don't send a new PR - just commit to your already existing branch. It will update this PR. 😉

Reply:
Done